test_helpers_end_to_end/src/steps.rs

```diff
@@ -174,6 +174,11 @@ pub enum Step {
     /// know about the ingester, so the test needs to ask the ingester directly.
     WaitForPersistedAccordingToIngester,
 
+    /// set the namespace retention interval to a retention period,
+    /// specified in ns.  None represents infinite duration
+    /// (i.e. never drop data).
```

Comment:
Same sorts of nits here

```suggestion
    /// Set the namespace retention interval to a retention period,
    /// specified in ns relative to `now()`.  `None` represents infinite retention
    /// (i.e. never drop data).
```

Reply:
Thanks @carols10cents